paper-radio-group.html

```diff
@@ -26,94 +30,135 @@
 
 @group Paper Elements
 @element paper-radio-group
-@extends core-selector
-@homepage github.io
 -->
 
-<link rel="import" href="../core-a11y-keys/core-a11y-keys.html">
-<link rel="import" href="../core-selector/core-selector.html">
-<link rel="import" href="../paper-radio-button/paper-radio-button.html">
+<dom-module name="paper-radio-group">
+  <style>
+    :host {
+      display: inline-block;
+    }
 
-<polymer-element name="paper-radio-group" extends="core-selector" role="radiogroup">
+    iron-selector ::content > * {
+      padding: 12px;
+    }
+  </style>
 
   <template>
+    <iron-selector selected="{{selected}}" attr-for-selected="name"
+        selectable="paper-radio-button">
+      <content id="items" select="*"></content>
+    </iron-selector>
+  </template>
+
+</dom-module>
+
+<script>
+  Polymer({
+    is: 'paper-radio-group',
 
-    <core-a11y-keys target="{{}}" keys="up left" on-keys-pressed="{{selectPrevious}}"></core-a11y-keys>
-    <core-a11y-keys target="{{}}" keys="down right" on-keys-pressed="{{selectNext}}"></core-a11y-keys>
-  
-    <style>
-    
-      :host {
-        display: inline-block;
+    hostAttributes: {
+      role: 'radiogroup'
+    },
+
+    properties: {
+      selected: {
+        type: String,
+        notify: true,
+        reflectToAttribute: true,
+        observer: "_selectedChanged"
       }
-  
-      polyfill-next-selector { content: ':host > *'; }
-      ::content > * {
-        padding: 12px;
+    },
+
+    listeners: {
+      // TODO: this can be replaced by <iron-a11y-keys> when it's ready.
+      keydown: '_onKeyDown'
+    },
+
+    _selectedChanged: function() {
+      // TODO: This only needs to be async while a domReady event is unavailable.
+      this.async(function() {
+        this._selectIndex(this._valueToIndex(this.items, this.selected));
+      });
+    },
+
+    _onKeyDown: function(event) {
+      if (event.keyCode === 37 || event.keyCode == 38) {
```

Comment:
I don't have a strong opinion, but I'm leaning towards laziness :(
Ideally I'd move them to an uber "helpful things 4 all" repo, which doesn't exist, and starting one for 6 constants (+space and enter) is kind of really annoying. And if it's just constants for this file, then maybe it's overkill since I've left a comment to what they represent.

I can do it if you feel strongly about it though! :)